lms/templates/courseware/program_marketing.html

```diff
@@ -1,3 +1,404 @@
+        </h2>
+        <div class="instructor-profiles">
+          <% index = 0 %>
+          % for instructor in instructors:
+            <% index += 1 %>
+            <div class="profile-item profile-item-desktop">
+              <div class="avatar">
+                <div class="img-holder">
+                  <a href="#instructor-details-${index}" class="instructor-image">
+                    <img src="${static.url(instructor.get('image'))}"
+                         alt="${instructor.get('name')}"/>
+                  </a>
+                </div>
+              </div>
+              <div class="details">
+                <div class="name">
+                  <a href="#instructor-details-${index}" class="instructor-label">${instructor.get('name')}</a>
+                </div>
+                <div class="dept">${instructor.get('organization')}</div>
+              </div>
+
+              <div class="modal modal-custom" id="instructor-details-${index}">
+                <a href="#" class="focusKeeper"></a>
+                <div class="btn-close modal_close"><a href="#"><i class="fa fa-close"></i></a></div>
+                <div class="modal-body">
+                  <div class="modal-header">
+                    <div class="instructor-data">
+                      <div class="thumbnail">
+                        <img src="${instructor['image']}" alt="${instructor.get('name')}"/>
+                      </div>
+                      <h3>
+                        <span class="instructor-name">${instructor.get('name')}</span>
+                        <span>
+                          ${Text(_('{position} at {organization}')).format(
+                            position=instructor.get('title'),
+                            organization=instructor.get('organization')
+                          )}
+                        </span>
+                      </h3>
+                    </div>
+                  </div>
+                  <div class="instructor-bio">${instructor['bio']}</div>
+                </div>
+                <a href="#" class="focusKeeper"></a>
+              </div>
+            </div>
+          % endfor
+
+          <div class="pagination hidden-xs">
+            <div class="pages">
+              ## Translators: Pagination range contains ordinal numbers of the first and the last instructor
+              ## whose information is shown on the current page.
+              ${Text(_("{pagination_range} of {number_of_instructors}")).format(
+                pagination_range=HTML('<span class="pagination-start"></span>-<span class="pagination-end"></span>'),
+                number_of_instructors=HTML('<span class="instructor-size">{number_of_instructors}</span>').format(
+                  number_of_instructors=len(instructors)
+                )
+              )}
+            </div>
+            <div class="controls">
+              <a href="#" id="pagination-previous"><span class="fa fa-chevron-up"></span></a>
```

Comment:
I think there should a screen reader element explaining what these icons are for.